twilight-model/src/poll/answer.rs

```diff
@@ -0,0 +1,75 @@
+use super::media::PollMedia;
+use serde::{Deserialize, Serialize};
+
+/// A poll answer.
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct PollAnswer {
+    /// The ID of the answer.
+    ///
+    /// This is unique within the poll. And increases
+    /// sequentially with each answer. Currently, only
+    /// 1-10 answers are allowed.
+    pub answer_id: u8,
+    /// The data of the answer.
+    pub poll_media: PollMedia,
```

Comment:
It seems like this one is left out from http responses, but it is not used in any of the http responses?? 

I think no changes are needed here, but I wanted to highlight it.